CHANGELOG.md

```diff
@@ -171,6 +171,30 @@ In particular, `num_args` (the replacement for `takes_value`) will default appro
 from the `ArgAction` and generally only needs to be set explicitly for the
 other `num_args` use cases.
 
+#### Migrating Derive
+- Rename `clap` to `command` or `arg`
```

Comment:
This is not a required change for migrating.  `clap` should still work, its just deprecated.  If you build with `cargo check -F clap/deprecated`, it should point out which attribute a `clap` attribute should be replaced with